cvat/apps/dataset_manager/annotation.py

```diff
@@ -930,7 +954,7 @@ def propagate(shape, end_frame, *, included_frames=None):
         prev_shape = None
         for shape in sorted(track["shapes"], key=lambda shape: shape["frame"]):
             curr_frame = shape["frame"]
-            if included_frames is not None and curr_frame not in included_frames:
+            if deleted_frames is not None and curr_frame in deleted_frames:
```

Comment:
since `deleted_frames` is not propagated more to other functions/methods, perhaps we can use `deleted_frames = deleted_frames or []` and exclude one check from if-statements here and in 1009 lines.

Reply:
Ok